.yamato/Collate Builds.yml

```diff
@@ -0,0 +1,37 @@
+name: Collate Builds
+
+agent:
+  type: Unity::VM
+  image: cds-ops/ubuntu-18.04-agent:v1.0.11-765607
+  flavor: b1.large
+
+dependencies:
+  - .yamato/Build Android.yml
+  - .yamato/Build Linux x64.yml
+  - .yamato/Build Linux x86.yml
+  - .yamato/Build Classlibs OSX.yml
+  - .yamato/Build Runtime OSX.yml
+  - .yamato/Build Windows x64.yml
+  - .yamato/Build Windows x86.yml
+  
+commands:
+  - .yamato/scripts/collate_builds.sh
+
+triggers:
+  branches:
+    only:
+      - "/.*/"
```

Comment:
I was going to update our mono scripts. It seems like we run on all branches, all pushes, even with no PRs? Is that correct? We likely want to make it manual except for PRs to unity-master?

Reply:
Done